code/modules/reagents/chemistry/reagents/drinks/drink_reagents.dm

```diff
@@ -1321,7 +1321,6 @@
 	color = "#DFDFDF" // rgb: 223, 223, 223
 	taste_description = "ayran"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
-	default_container = /obj/item/reagent_containers/cup/glass/ayran
```

Comment:
bunu kaldırmak yerine yeni eklediğin bottle yapsan iyi olur yoksa böyle gözüküyo
![image](https://github.com/user-attachments/assets/547a34df-76db-404b-8e41-5d5e6de4fda7)